src/aosm/azext_aosm/generate_nsd/templates/nsd_template.bicep

```diff
@@ -75,7 +75,7 @@ resource nsdVersion 'Microsoft.Hybridnetwork/publishers/networkservicedesigngrou
     nfvisFromSite: {
       nfvi1: {
         name: nfviSiteName
-        type: '{{nfviSiteType}}'
+        type: 'AzureCore'
```

Comment:
why are we hardcoding this?

Reply:
Jamie didn't want to have this as a parameter. It is this type in majority of cases so I think it is ok